config/canonical.json

```diff
@@ -22032,8 +22032,11 @@
   },
   "shop/convenience|VinMart+": {
     "count": 538,
+    "countryCodes": ["vn"],
     "tags": {
       "brand": "VinMart+",
```

Comment:
On further consideration I think this should be `brand=VinMart` and then sharing the same wikidata makes sense.

Reply:
Yeah we are a bit inconsistent in the index about this..  I could see it being done either way, but I'm leaning towards having both entries with `brand=VinMart` and same `brand:wikidata=Q60245505`, but different `name` and `shop` tags.

(I looked at Tesco / Tesco Express for an example of a brand with both supermarket and convenience store -but now I'm pretty sure that one is currently done incorrectly 😢).

Reply:
I switched it to `brand=VinMart` in 5cb90e0

Reply:
I think we should off a separate Wikidata item for VinMart+ and change the `brand` back. There are plenty of precedents in the index for qualifiers in brand names, from Costco Pharmacy to Shell Select.

Reply:
> I think we should off a separate Wikidata item for VinMart+

Done in 950885a46edcad9fd4d12d1f89749cc7ad05ebd9.